test/commands/scanner/rule/remove.test.ts

```diff
@@ -37,154 +30,108 @@ const customPathDescriptor = {
 	}
 };
 
-const removeTest = setupCommandTest
-	.do(() => {
+describe('scanner rule remove', () => {
+	beforeEach(() => {
+		TestOverrides.initializeTestSetup();
```

Comment:
Since the tests start by creating a `CustomPaths.json`, we need to call this method immediately instead of waiting to do it in `runCommand`.